magellan-library/src/main/java/com/wealthfront/magellan/view/DialogComponent.kt

```diff
@@ -1,12 +1,12 @@
 package com.wealthfront.magellan.view
 
+import android.app.Activity
 import android.app.Dialog
 import android.content.Context
+import com.wealthfront.magellan.DialogCreator
 import com.wealthfront.magellan.lifecycle.LifecycleAware
 import javax.inject.Inject
 
-typealias DialogCreator = (Context) -> Dialog
```

Comment:
We want to use the Java interface for java interop? Could we overload the method so we can use nice Kotlin syntax without a SAM conversion for our new `Step`s?

Reply:
Overloaded the `showDialog` method.